src/django_components/__init__.py

```diff
@@ -6,6 +6,15 @@
 # isort: off
 from django_components.app_settings import ContextBehavior, ComponentsSettings
 from django_components.autodiscovery import autodiscover, import_libraries
+from django_components.util.command import (
+    CommandArg,
+    CommandArgGroup,
+    CommandHandler,
+    CommandLiteralAction,
+    CommandParserInput,
+    CommandSubcommand,
+    ComponentCommand,
```

Comment:
Exported the command-related entities. `ComponentCommand` is the command class. The rest of the `CommandXxx` variables are for typing.